### ISSUE TITLE
Find duplicate artists in a playlist (feature request)

### ISSUE THREAD

**Issue by 45Roller** [JMPerez/spotify-dedup#54]
Would it be possible to find duplicate artists in larger playlists?

**Comment by gianpaj** [comment]
What do you mean? One song per artist in each playlist? or album of an artist?

**Comment by 45Roller** [comment]
Yeah one song per artist, as I want to discover new artists all the time

**Comment by 45Roller** [comment]
can this be done? should be pretty straightforward for those who know, moving the duplicate lookup from the track table to the artist one?
